specifyweb/frontend/js_src/lib/components/Formatters/Preview.tsx

```diff
@@ -36,7 +36,7 @@ export function useResourcePreview(table: SpecifyTable): {
           table.name,
           {
             limit: defaultPreviewSize,
-            domainFilter: true,
+            domainFilter: false, // set to true after scoping reimplementation
```

Comment:
can this be turned into a todo comment so that it's less likely to get lost?
as per https://github.com/specify/specify7/tree/production/specifyweb/frontend/js_src/lib#todos, BUG or REFACTOR types work best here

```suggestion
            domainFilter: false, // REFACTOR: set to true after scoping reimplementation
```

Reply:
ya, that's a good idea